modules/swagger-maven-plugin/src/main/java/io/swagger/v3/plugin/maven/SwaggerMojo.java

```diff
@@ -191,19 +334,38 @@ public void execute() throws MojoExecutionException, MojoFailureException
     @Parameter( property = "resolve.openapiFilePath")
     private String openapiFilePath;
 
+    /**
+     * @since 2.0.8
+     */
+    @Parameter(property = "resolve.configurationFilePath")
+    private String configurationFilePath;
+
     @Parameter(defaultValue = "${project}", readonly = true)
     private MavenProject project;
 
     @Parameter( property = "resolve.encoding" )
     private String encoding;
 
     private String projectEncoding = "UTF-8";
+    private SwaggerConfiguration config;
 
     public String getOutputPath() {
         return outputPath;
     }
+
     public String getOpenapiFilePath() {
         return openapiFilePath;
     }
 
+    String getConfigurationFilePath() {
+        return configurationFilePath;
+    }
+
+    void setContextId(String contextId) {
```

Comment:
is this just for tests?

Reply:
Correct. That is why I left it with package visibility. Do you have a suggestion to better mark it as test method (Im familiar with a `@VisibleForTesting` annotation found in `com.google.common.annotations`)?